components/home/sake/SakeCelebration.tsx

```diff
@@ -61,17 +61,15 @@ const SakeCelebration: StylableFC = ({ style, className }) => {
       <Image
         src={SakeImage}
         alt={t("headshotAlt")}
-        className="absolute right-0 top-0 h-64 w-auto"
+        className="absolute right-0 top-8 h-64 w-auto sm:top-0"
       />
       <div
-        className={cn(`mt-64 flex h-full flex-col rounded-tl-[inherit]
+        className={cn(`relative mt-64 flex h-full flex-col rounded-t-[inherit]
```

Comment:
This should resolve the uneven rounded corners, an absolutely groundbreaking, life-changing change! Could not live with that.